### PR TITLE
Fix links in documentation

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
I have planned to fix all links in documentation.

### CODE REVIEW

**Comment by HolgerJeromin** [comment]
ref https://github.com/streetcomplete/StreetComplete/commit/6fe3387783d7de0ce283ff44186e735277912778

File: CONTRIBUTING_A_NEW_QUEST.md [hunk 1]
```diff
@@ -509,7 +509,7 @@ It is possible to check for [age of elements](app/src/main/java/de/westnordost/s
 
 It is possible to share and reuse [information about tagging schemes](app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddRoadSurface.kt#L18).
 
-(this info is gathered [here](/app/src/main/java/de/streetcomplete/StreetComplete/data/meta/OsmTaggings.kt))
+(this info is gathered [here](/app/src/main/java/de/westnordost/streetcomplete/osm))
```
Comment (mnalis) [inline CONTRIBUTING_A_NEW_QUEST.md:512]:
```suggestion
(this info is gathered [here](app/src/main/java/de/westnordost/streetcomplete/osm))
```

(use relative links like elsewhere, i.e. without leading `/`)

**Comment by Jean-BaptisteC** [comment]
ref https://github.com/streetcomplete/StreetComplete/commit/5a2286bd8140ec5ae958a76a46af27ee24d5fbd4

**Comment by Jean-BaptisteC** [comment]
All links in documentation has been tested

**Comment by matkoniecz** [comment]
At some point I will get that dead link detector running... (I started working on it and run into bugs in libraries I was planning to use - I should look whether they fixed them or just write spider + error checker myself)

**Comment by mnalis** [comment]
> At some point I will get that dead link detector running...

Something like this (have not tried)?
https://andrewwegner.com/find-broken-links-with-github-actions.html

**Comment by matkoniecz** [comment]
> I use [broken-link-checker](https://www.npmjs.com/package/broken-link-checker) for now. I picked this because I'm familiar with it. It hasn't been updated since 2018, though, so in the future I may spend some time either looking for an alternative or building an alternative. For the time being though, it works just fine.

Well, for me it is not working fine - see https://github.com/stevenvachon/broken-link-checker/issues/created_by/matkoniecz (it is one of projects I tried)

**Pull request merged**